### DISCUSSION TITLE
Poly sync not working properly for projects in sub-directories

### DISCUSSION THREAD

**Discussion by olafurnielsen** [DavidVujic/python-polylith#185] [Ideas]
**Describe the bug**

- I'd like to be able to organize projects in sub-directories under the projects folder
- poetry poly info correctly finds all projects, even those in nested directories
- I can run poetry poly sync to have bricks added to pyproject.toml
- But the generated path is always `../../bases`

**To Reproduce**
1. Create a project in a sub-directory under projects
2. Run poetry poly sync
3. Run poetry build-project within the respective project

**Expected behavior**
Relative paths to components/bases should be correct – f.x. `../../../bases` for a project residing in `projects/sub-directory/project_a`

**Desktop (please complete the following information):**
 - OS: Mac OS 14.3.1
 - Python version: 3.12.2
 - Poetry version: 1.8.2

**Comment by DavidVujic** [comment]
Hi @olafurnielsen, thank you for submitting this issue!

The suggested structure is to have the projects folder at one level. The `poly info` command finds it because that is how the file lookup is currently developed, but that was not intended (more like by accident 😄).

I think you can structure this as you like (even if it is not recommended), but without using the `poly sync` command. 

I will move this issue to the discussions 💡 section!
